src/modules/person/persistence/person.repository.ts

```diff
@@ -95,6 +127,38 @@ export class PersonRepository {
         return null;
     }
 
+    public async getPersonIfAllowed(personId: string, permissions: PersonPermissions): Promise<Result<Person<true>>> {
+        const scope: PersonScope = await this.getPersonScopeWithPermissions(permissions);
+        scope.findBy({ id: personId }).sortBy('vorname', ScopeOrder.ASC);
+
+        const [persons]: Counted<Person<true>> = await this.findBy(scope);
+        const person: Person<true> | undefined = persons[0];
+
+        if (!person) return { ok: false, error: new EntityNotFoundError('Person') };
+
+        return { ok: true, value: person };
+    }
+
+    public async deletePersonIfAllowed(
```

Comment:
"IfAllowed" should not be part of the name, because every repo method should only work if the user is allowed.

Reply:
Because the method checks for permission to read the user before deletion so I thought it made sense. But you are right, maybe it's 'redundant' to mention it again.